.github/workflows/swift-toolchain.yml

```diff
@@ -793,12 +793,22 @@ jobs:
           path: |
             ${{ github.workspace }}/BinaryCache/1/**/*.pdb
 
-      - uses: microsoft/action-publish-symbols@v2.1.6
+      - name: Upload PDBs to Azure
```

Comment:
This is in the compilers job. I believe we need this in the sdk job (or both)

Reply:
Yes, we will want it in both. I'd really like to try to get the glob working though.

Reply:
Pushed a fix.